Lib/test/test_tokenize.py

```diff
@@ -1862,7 +1862,7 @@ def test_random_files(self):
         # TODO: Remove this once we can untokenize PEP 701 syntax
         testfiles.remove(os.path.join(tempdir, "test_fstring.py"))
 
-        for f in ('buffer', 'builtin', 'fileio', 'inspect', 'os', 'platform', 'sys'):
```

Comment:
You don't have to change this code to look in parent directory?

Reply:
No :)

This module was excluded from this test - explicitly here. But, since now it is not a module, but a package - we can safely remove the ignore rule here.

Reply:
Ah wait, this file is not moved into test_inspect/, we are good, ok. Nevermind.